extmod/machine_spi.c

```diff
@@ -227,6 +221,10 @@ static void mp_machine_soft_spi_init(mp_obj_base_t *self_in, size_t n_args, cons
     if (args[ARG_phase].u_int != -1) {
         self->spi.phase = args[ARG_phase].u_int;
     }
+    if (args[ARG_firstbit].u_int != -1) {
+        self->spi.firstbit = args[ARG_firstbit].u_int == MICROPY_PY_MACHINE_SPI_MSB ?
+            MICROPY_PY_MACHINE_SPI_MSB : MICROPY_PY_MACHINE_SPI_LSB;
```

Comment:
As above, no need to validate the value here.

Reply:
Done.